packages/flutter/test/material/app_bar_test.dart

```diff
@@ -2563,6 +2563,223 @@ void main() {
       expect(getAppBarBackgroundColor(tester), defaultColor);
       expect(tester.getSize(findAppBarMaterial()).height, kToolbarHeight);
     });
+
+    testWidgets('Material3 - scrolledUnderElevation should be maintained when drawer is opened', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('scrolledUnderElevation should be maintained when drawer is opened', (WidgetTester tester) async {
```

Material 3 is default now